includes/setup.inc.sh

```diff
@@ -1,9 +1,27 @@
 #!/usr/bin/env bash
-set -x
 
-if ! command -v mvn3 >/dev/null; then
-  alias mvn3='mvn '
-  function mvn3() {
-      mvn "${@}"
-  }
+function __debug() {
+  # echo ${@}
+  true
+}
+
+if type brew &>/dev/null; then
+  __debug "Detected Homebrew" 
+  # Find maven-snapshot installation directory
+  mvn4dir=$(find /opt/homebrew/Cellar/maven-snapshot -type d -name "4*")
+  if [ -f $mvn4dir/bin/mvn ] && [ -x $mvn4dir/bin/mvn ]; then
+    __debug Found Maven 4 and it is executable: $($mvn4dir/bin/mvn --version | head -n 1)
+    function mvn4() {
+      $mvn4dir/bin/mvn "${@}"
+    }
+  fi
+
+  # Find maven installation directory
+  mvn3dir=$(find /opt/homebrew/Cellar/maven -type d -name "3*")
+  if [ -f $mvn3dir/bin/mvn ] && [ -x $mvn3dir/bin/mvn ]; then
+    __debug Found Maven 3 and it is executable: $($mvn3dir/bin/mvn --version | head -n 1)
+    function mvn3() {
+      $mvn3dir/bin/mvn "${@}"
+    }
+  fi
```

Comment:
Use `${$HOMEBREW_PREFIX:-/opt/homebrew}`. I usually install homebrew software in `$USER/.local/homebrew/`…